### DISCUSSION TITLE
大佬，咨询一个比较小白的问题

### DISCUSSION THREAD

**Discussion by zqlit** [usememos/memos#18] [Q&A] [answered]
我使用的是宝塔面板，已经将memos.db下载至本地
[![7l8N7Q.png](https://s4.ax1x.com/2022/01/13/7l8N7Q.png)](https://imgtu.com/i/7l8N7Q)
使用docker部署命令docker run --rm --pull always --name memos -p 8080:8080 -v ~/data/:/usr/local/memos/data/ -d neosmemo/memos，部署之后，当我重启了服务器了之后，数据全部丢失，请问有什么方法可以解决这个问题

**Comment by boojack** [comment, marked as answer]
你的 `memos.db` 在 `根目录 > data`，也就是 `/data/memos.db`，所以部署命令应该是
```
docker run --rm --pull always --name memos -p 8080:8080 -v /data/:/usr/local/memos/data/ -d neosmemo/memos
```

**Reply by zqlit** [reply]
谢谢大佬 解决了

**Reply by rebron1900** [reply]
一个是绝对路径一个是相对路径

**Comment by MaoVlog** [comment]
宝塔如何用docker如何搭建！能给详细的教程嘛！我小白一个！😅

**Reply by boojack** [reply]
如果有云服务器的话，不需要额外安装宝塔，可以直接在装好 docker 后，通过 readme 里的一行代码执行就能跑起来了

**Reply by MaoVlog** [reply]
好的谢谢~
